src/igvfd/schemas/tabular_file.json

```diff
@@ -146,6 +147,8 @@
                 "sequence barcodes",
                 "tissue positions",
                 "transcript quantifications",
+                "unfiltered global differential expression",
```

Comment:
1. The tabular_file changelog is not updated accordingly.
2. Analysis step changelog for input_content_type and output_content_type should also be updated.

Reply:
done